README.md

```diff
@@ -1,196 +0,0 @@
-# Intro to Reducers
-
-## Objectives
-1. Describe how a reducer takes in an action and returns updated state
-2. Determine why reducers must be pure functions
-3. Organize reducers more cleanly using reducer composition
-
-## You've got your actions ... now what?
-
-As we discussed in the previous lab, Redux holds the state for our whole application. Actions contain data and are sent to a store using the dispatch function. Our actions are JavaScript objects that send data to the store.
-
-```javascript
-{
-    type: ADD_TODO,
-    text: 'Learn about reducers'
-}
-
-{
-    type: ADD_MIXER,
-    text: 'vermouth'
-}
-```
-
-Actions simply convey that something happened, they don't tell the store how to update state based on those actions. **That's where reducers come in.**
-
-## Reducers
-
-A reducer is a pure function that takes in the previous state and the action and returns the next state.
-
-What's a pure function? No, it's not a function that's gone organic, vegan, and gluten free.
-
-A pure function:
-- Has no side effects
-- Has no mutations
-- Its return value depends entirely on what is passed in
-- It doesn't modify values passed do it
-
-Pure functions don't make calls to an API, change the database or alter the arguments passed in. Given the same arguments, it would return the same value every time.
-
-```javascript
-const square = (x) => {
-  return x^2
-}
-```
-vs
-```javascript
-const updateDatabase = (x, y) => {
-  x.name = y
-}
-```
-
-Since Redux calls our reducers with an undefined state for the first time, we have an opportunity to define the initial state of our app. Using ES6 default arguments syntax, we set our initial state to an empty array.
```

Comment:
point out that we use an empty array for lists of items - we could use an object or initial value if we wanted.